packages/cms-cli/commands/hubdb.js

```diff
@@ -27,6 +30,11 @@ const {
 } = require('../lib/commonOpts');
 const { logDebugInfo } = require('../lib/debugInfo');
 
+const CREATE_COMMAND_NAME = 'hubdb create';
+const FETCH_COMMAND_NAME = 'hubdb fetch';
+const CLEAR_COMMAND_NAME = 'hubdb clear';
+const DELETE_COMMAND_NAME = 'hubdb delete';
```

Comment:
Since we use hyphens (e.g. `hubdb-create`) for the filemanager subcommands, i think we should do the same here. I also think these constants are a little overkill since we're only using them in one place.

Reply:
updated